### DISCUSSION TITLE
Is there any way to set up your own import snapshot data directory?

### DISCUSSION THREAD

**Discussion by coder-chenda** [filecoin-project/lotus#8595] [Build and Sync] [answered]
I'd like to ask how to set the path of import snapshot to import data, because I already have data path L, and the default import will be / home / Ubuntu / Lotus /. I'm very annoyed. Is there any way to set your own import snapshot data directory?

**Comment by rjan90** [comment, marked as answer]
Hey @coder-chenda! Can you explain what you mean about `set your own import snapshot data directory`? 

You can set where your .lotus-folder is located by setting the `LOTUS_PATH=/path/to/your/.lotus` environmental variable to your desired location if that is what you are wondering about?

**Reply by coder-chenda** [reply]
Hello, I set it, but it's useless

**Reply by coder-chenda** [reply]
The storage path on my machine is /data/fil/data/

**Reply by rjan90** [reply]
You will have to export that variable before you start your deamon. So if your daemon is already running, you will have to restart it for it to update the location of the `.lotus`-folder

`export  LOTUS_PATH=/data/fil/data/.lotus`

**Reply by coder-chenda** [reply]
Yes, I export variables first and then start lotus daemon

**Reply by coder-chenda** [reply]
But the data is still exported to /root/.lotus

**Comment by LexLuthr** [comment]
@coder-chenda Are you running it as a service from root user? If so, then you need to set these variables in the service file.

**Comment by LexLuthr** [comment]
I am archiving this discussion as there has been no new input from the author for more than 7 days.
